targets/ath79-generic

```diff
@@ -131,6 +131,12 @@ device('tp-link-eap225-outdoor-v1', 'tplink_eap225-outdoor-v1', {
        packages = ATH10K_PACKAGES_QCA9888,
 })
 
+device('tp-link-tl-wr1043nd-v3', 'tplink_tl-wr1043nd-v3', {
+	manifest_aliases = {
+		'tp-link-tl-wr1043n-nd-v3',
```

Comment:
Missing "upgrade from 19.07" comment.

Reply:
resolved in eaea64c905bb7c8c56925847d3f86fb8d71d751b